lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -10,18 +24,16 @@
 
```

Comment:
So, I have an issue with the DOM order here.  The instructions come after the camera control.  As a screen reader user, I would encounter the camera control before receiving the instructions on how to use it.  I suggest swapping the columns or using float to swap the order in the DOM, but keep the visual rendering order.